app-admin/u-boot-menu/autobuild/build

```diff
@@ -0,0 +1,11 @@
+mkdir -p "$PKGDIR"/usr/share/u-boot-menu/conf.d
```

Comment:
These mkdir's are not necessary, the commands below would create them.

Reply:
In fact, if this line is removed, it will cause the packaging process to fail because, during the build process, there is a step that runs `install -Dm644 "$SRCDIR"/read-config "$PKGDIR"/usr/share/u-boot-menu`.

Reply:
Changed to `mkdir -p "$PKGDIR"/usr/share/u-boot-menu`